### PR TITLE
added three pages in properties API :- Index (get all properties ), S…

### PR DESCRIPTION
**Pull request by chopraapooja**
…how (get 1 property ), history (self-explanatry)

### CODE REVIEW

File: source/includes/properties/_01-show.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## Get one property
+
+```shell
+$ curl 'http://ci.example.com/go/properties/PipelineName/541/StageName/1/JobName'/cruise_agent \
```
Comment (ketan) [inline source/includes/properties/_01-show.md]: The `/cruise_agent` is outside the quote.

File: source/includes/properties/_02-history.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+myLocalAgent,0,13,Passed,8,4f9e580347b2e259fe030a775771359cdc984346,1,2015-07-07T09:44:27+05:30,2015-07-07T09:44:34+05:30,2015-07-07T09:44:44+05:30,2015-07-07T09:44:46+05:30,2015-07-07T09:44:46+05:30,2015-07-07T09:44:46+05:30
+
+myLocalAgent,0,14,Passed,9,4f9e580347b2e259fe030a775771359cdc984346,1,2015-07-07T10:17:37+05:30,2015-07-07T10:17:45+05:30,2015-07-07T10:17:55+05:30,2015-07-07T10:17:56+05:30,2015-07-07T10:17:56+05:30,2015-07-07T10:17:56+05:30
+```
+
+
+### HTTP Request
+
+`GET /go/properties/search?pipelineName=[PipelineName]&stageName=[StageName]&jobName=[JobName]&limitPipeline=[PipelineCounter]&limitCount=[Number]`
```
Comment (ketan) [inline source/includes/properties/_02-history.md]: This does not follow the usual convention that we have.

**Comment by ketan** [comment]
There are a few improvements that can be done, I'm going to make them myself, but please make sure to review them to apply those in the future.

**Comment by ketan** [comment]
Hey @chopraapooja, can you setup redirects from the old site as such - https://github.com/gocd/documentation/blob/master/user/api/users_api.md. That way we know it's been migrated.

**Comment by chopraapooja** [comment]
[done] (https://github.com/gocd/documentation/pull/156)

**Pull request merged**